tdrs-backend/tdpservice/settings/common.py

```diff
@@ -329,24 +321,17 @@ class Common(Configuration):
 
     s3_src = "s3-us-gov-west-1.amazonaws.com"
 
-    CSP_DEFAULT_SRC = ("'self'", "*.cloud.gov",  "*.acf.hhs.gov")
+    CSP_DEFAULT_SRC = ("'none'")
     CSP_SCRIPT_SRC = ("'self'", s3_src)
     CSP_IMG_SRC = ("'self'", "data:", s3_src)
     CSP_FONT_SRC = ("'self'", s3_src)
-    CSP_CONNECT_SRC = ("'self'", "*.cloud.gov", "*.acf.hhs.gov")
+    CSP_CONNECT_SRC = ("'self'", "*.cloud.gov")
     CSP_MANIFEST_SRC = ("'self'")
     CSP_OBJECT_SRC = ("'none'")
     CSP_FRAME_ANCESTORS = ("'none'")
     CSP_FORM_ACTION = ("'self'")
     CSP_STYLE_SRC = ("'self'", s3_src, "'unsafe-inline'")
-    CORS_ALLOW_HEADERS = (
-        'x-requested-with',
-        'content-type',
-        'accept',
-        'origin',
-        'authorization',
-        'X-CSRFToken'
-    )
```

Comment:
This might never have been needed here. Might also have been fixed in NGINX config?

Reply:
this is the default value for `CORS_ALLOW_HEADERS`, according to the `django-cors-headers` docs
```
CORS_ALLOW_HEADERS = [
    "accept",
    "accept-encoding",
    "authorization",
    "content-type",
    "dnt",
    "origin",
    "user-agent",
    "x-csrftoken",
    "x-requested-with",
]
```
https://pypi.org/project/django-cors-headers/

Reply:
Yup, that's fine, we can leave this question for our techdebt documentation ticket #2031